src/index.html

```diff
@@ -11,12 +11,67 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,400&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="header-logo">
+        <a href="/">
+          <img
+            src="images/logo.png"
+            alt="Moyo logo"
+          />
+        </a>
+      </div>
+      <nav class="navigation">
+        <ul class="nav-link-list">
+          <li class="link-item">
+            <a href="/">Apple</a>
+          </li>
+          <li class="link-item">
+            <a href="/">Samsung</a>
+          </li>
+          <li class="link-item">
+            <a href="/">Smartphones</a>
+          </li>
+          <li class="link-item">
+            <a
+              href="/"
+              data-qa="hover"
+              class="is-active"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="link-item">
+            <a href="/">Gadgets</a>
+          </li>
+          <li class="link-item">
+            <a href="/">Tablets</a>
+          </li>
+          <li class="link-item">
+            <a href="/">Photo</a>
+          </li>
+          <li class="link-item">
+            <a href="/">Video</a>
```

Comment:
The `href` attributes for navigation links and the logo are currently set to `/`. If these are meant to navigate to specific sections or the homepage, they should be updated with the correct URLs to ensure proper navigation.